doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -147,6 +147,8 @@ Bluetooth samples
 
 * Added :ref:`peripheral_ams_client` sample.
 
+* Removed alexa_gadgets Bluetooth sample due to Amazon pausing support for the Gadgets ecosystem.
```

Comment:
```suggestion
* Removed Peripheral Alexa Gadgets Bluetooth sample due to Amazon pausing support for the Gadgets ecosystem.
```
Please add also the removal of the Alexa Gadgets service library.

Reply:
Done. Reference to the library in the old release notes was removed already, as this caused a doc build failure.